test/members_page_test.rb

```diff
@@ -0,0 +1,50 @@
+# frozen_string_literal: true
+require_relative './test_helper'
+require_relative '../lib/members_page.rb'
+
+describe 'MembersPage' do
+  around { |test| VCR.use_cassette('Members Page', &test) }
+
+  let :members_page do
+    url = 'http://sitl.diputados.gob.mx/LXIII_leg/listado_diputados_gpnp.php?tipot=TOTAL'
+    MembersPage.new(response: Scraped::Request.new(url: url).response)
+  end
+
+  describe 'party groupings' do
+    subject do
+      members_page.party_groupings
+                  .first
```

Comment:
There doesn't really seem to be much value in splitting this over multiple lines